rq/cli/helpers.py

```diff
@@ -198,6 +202,82 @@ def setup_loghandlers_from_args(verbose, quiet, date_format, log_format):
     setup_loghandlers(level, date_format=date_format, log_format=log_format)
 
 
+def job_func(func, args, kwargs):
+    try:
+        importlib.import_module(func)  # execute file
+    except ImportError:
+        return import_attribute(func)(*args, **kwargs)
+
+
+def parse_function_arg(argument):
+    mode = 0  # 0: text, 1: json, 2: literal_eval (python)
+    keyword = None
+    if argument.startswith(':'):  # no keyword, json
+        mode = 1
+        value = argument[1:]
+    elif argument.startswith('#'):  # no keyword, literal_eval
+        mode = 2
+        value = argument[1:]
+    else:
+        index = argument.find('=')
+        if index > 0:
+            if ':' in argument and argument.index(':') + 1 == index:  # keyword, json
+                mode = 1
+                keyword = argument[:index - 1]
+            elif '#' in argument and argument.index('#') + 1 == index:  # keyword, literal_eval
+                mode = 2
+                keyword = argument[:index - 1]
+            else:  # keyword, text
+                keyword = argument[:index]
+            value = argument[index + 1:]
+        else:  # no keyword, text
+            value = argument
+
+    if value.startswith('@'):
+        try:
+            with open(value[1:], 'r') as file:
+                value = file.read()
+        except FileNotFoundError:
+            raise click.FileError(value[1:], 'Not found')
+
+    if mode == 1:  # json
+        value = loads(value)
+    elif mode == 2:  # literal_eval
+        value = literal_eval(value)
+
+    return keyword, value
+
+
+def parse_function_args(arguments):
+    args = []
+    kwargs = {}
+
+    for argument in arguments:
+        try:
+            keyword, value = parse_function_arg(argument)
+        except JSONDecodeError:
+            raise click.BadParameter('Couldn\'t parse json.')
+        except (ValueError, SyntaxError):
+            raise click.BadParameter(
+                'Could not eval. View: https://docs.python.org/3/library/ast.html#ast.literal_eval')
```

Comment:
`Unable to eval <keyword> as Python object. See https://docs.python.org/3/library/ast.html#ast.literal_eval`